samples/highcharts/studies/data-layer/demo.mjs

```diff
@@ -0,0 +1,191 @@
+import '../../../../code/es-modules/Core/Renderer/SVG/SVGRenderer.js';
```

Comment:
This demo fails for me with JS errors...

Reply:
This demo is a bit outdated in a few ways, we have changed how the modifiers work, the DataSeriesConverter and the DataTable. So it might be simplest to retire this iteration and create a new demo for the series-dataTable integration when that is ready.